Fix new TOX errors

```
$ tox -e analysis
GLOB sdist-make: C:\Users\allmi\Source\softlayer-python\setup.py
analysis recreate: C:\Users\allmi\Source\softlayer-python\.tox\analysis
analysis installdeps: -rC:\Users\allmi\Source\softlayer-python/tools/test-requirements.txt, hacking, pylint
analysis inst: C:\Users\allmi\Source\softlayer-python\.tox\.tmp\package\1\SoftLayer-6.0.2.zip
analysis installed: alabaster==0.7.12,astroid==2.11.5,atomicwrites==1.4.0,attrs==21.4.0,Babel==2.10.1,certifi==2022.5.18.1,charset-normalizer==2.0.12,click==8.1.3,colorama==0.4.4,commonmark==0.9.1,coverage==6.4.1,coveralls==3.3.1,dill==0.3.5.1,distlib==0.3.4,docopt==0.6.2,docutils==0.18.1,filelock==3.7.1,flake8==3.8.4,hacking==4.1.0,idna==3.3,imagesize==1.3.0,iniconfig==1.1.1,isodate==0.6.1,isort==5.10.1,Jinja2==3.1.2,lazy-object-proxy==1.7.1,lxml==4.9.0,MarkupSafe==2.1.1,mccabe==0.6.1,mock==4.0.3,packaging==21.3,platformdirs==2.5.2,pluggy==1.0.0,prettytable==3.3.0,prompt-toolkit==3.0.29,py==1.11.0,pycodestyle==2.6.0,pyflakes==2.2.0,Pygments==2.12.0,pylint==2.14.1,pyparsing==3.0.9,pytest==7.1.2,pytest-cov==3.0.0,pytz==2022.1,requests==2.27.1,requests-file==1.5.1,requests-toolbelt==0.9.1,rich==12.3.0,six==1.16.0,snowballstemmer==2.2.0,SoftLayer @ file:///C:/Users/allmi/Source/softlayer-python/.tox/.tmp/package/1/SoftLayer-6.0.2.zip,softlayer-zeep==5.0.0,Sphinx==5.0.1,sphinxcontrib-applehelp==1.0.2,sphinxcontrib-devhelp==1.0.2,sphinxcontrib-htmlhelp==2.0.0,sphinxcontrib-jsmath==1.0.1,sphinxcontrib-qthelp==1.0.3,sphinxcontrib-serializinghtml==1.1.5,toml==0.10.2,tomli==2.0.1,tomlkit==0.11.0,tox==3.25.0,urllib3==1.26.9,virtualenv==20.14.1,wcwidth==0.2.5,wrapt==1.14.1
analysis run-test-pre: PYTHONHASHSEED='178'
analysis run-test: commands[0] | flake8 SoftLayer tests
analysis run-test: commands[1] | pylint SoftLayer -r n --ignore=tests,fixtures -d too-many-locals -d star-args -d redefined-variable-type -d locally-disabled -d no-else-return -d len-as-condition -d useless-object-inheritance -d consider-using-in -d consider-using-dict-comprehension -d useless-import-alias -d consider-using-f-string --max-args=25 --max-branches=20 --max-statements=65 --min-public-methods=0 --max-public-methods=35 --min-similarity-lines=30 --max-line-length=120
************* Module Command line
Command line:1:0: R0022: Useless option value for '-d', 'star-args' was removed from pylint, see https://pylint.pycqa.org/en/latest/whatsnew/1/1.4.html#what-s-new-in-pylint-1-4-3. (useless-option-value)
Command line:1:0: W0012: Unknown option value for '-d', expected a valid pylint message and got 'redefined-variable-type' (unknown-option-value)
************* Module SoftLayer.auth
SoftLayer\auth.py:8:0: W0012: Unknown option value for 'disable', expected a valid pylint message and got 'no-self-use' (unknown-option-value)
************* Module SoftLayer.CLI.environment
SoftLayer\CLI\environment.py:20:0: W0012: Unknown option value for 'disable', expected a valid pylint message and got 'no-self-use' (unknown-option-value)
************* Module SoftLayer.managers.account
SoftLayer\managers\account.py:15:0: W0012: Unknown option value for 'disable', expected a valid pylint message and got 'no-self-use' (unknown-option-value)
************* Module SoftLayer.managers.cdn
SoftLayer\managers\cdn.py:12:0: W0012: Unknown option value for 'disable', expected a valid pylint message and got 'no-self-use' (unknown-option-value)
************* Module SoftLayer.managers.dedicated_host
SoftLayer\managers\dedicated_host.py:16:0: W0012: Unknown option value for 'disable', expected a valid pylint message and got 'no-self-use' (unknown-option-value)
************* Module SoftLayer.managers.email
SoftLayer\managers\email.py:13:0: W0012: Unknown option value for 'disable', expected a valid pylint message and got 'no-self-use' (unknown-option-value)
************* Module SoftLayer.managers.hardware
SoftLayer\managers\hardware.py:23:0: W0012: Unknown option value for 'disable', expected a valid pylint message and got 'no-self-use' (unknown-option-value)
************* Module SoftLayer.managers.ordering
SoftLayer\managers\ordering.py:8:0: W0012: Unknown option value for 'disable', expected a valid pylint message and got 'no-self-use' (unknown-option-value)
************* Module SoftLayer.managers.storage
SoftLayer\managers\storage.py:28:0: W0012: Unknown option value for 'disable', expected a valid pylint message and got 'no-self-use' (unknown-option-value)
************* Module SoftLayer.managers.vs
SoftLayer\managers\vs.py:31:0: W0012: Unknown option value for 'disable', expected a valid pylint message and got 'no-self-use' (unknown-option-value)
SoftLayer\managers\vs.py:363:17: C2801: Unnecessarily calls dunder method __contains__. Use in keyword. (unnecessary-dunder-call)
SoftLayer\managers\vs.py:1046:19: C2801: Unnecessarily calls dunder method __contains__. Use in keyword. (unnecessary-dunder-call)
************* Module SoftLayer.managers.vs_capacity
SoftLayer\managers\vs_capacity.py:17:0: W0012: Unknown option value for 'disable', expected a valid pylint message and got 'no-self-use' (unknown-option-value)
************* Module SoftLayer.managers.vs_placement
SoftLayer\managers\vs_placement.py:14:0: W0012: Unknown option value for 'disable', expected a valid pylint message and got 'no-self-use' (unknown-option-value)
************* Module SoftLayer.transports.transport
SoftLayer\transports\transport.py:33:0: W0012: Unknown option value for 'disable', expected a valid pylint message and got 'no-self-use' (unknown-option-value)
************* Module SoftLayer.transports.__init__
SoftLayer\transports\__init__.py:20:0: W0012: Unknown option value for 'disable', expected a valid pylint message and got 'no-self-use' (unknown-option-value)

-----------------------------------
Your code has been rated at 9.99/10

ERROR: InvocationError for command 'C:\Users\allmi\Source\softlayer-python\.tox\analysis\Scripts\pylint.EXE' SoftLayer -r n --ignore=tests,fixtures -d too-many-locals -d star-args -d redefined-variable-type -d locally-disabled -d no-else-return -d len-as-condition -d useless-object-inheritance -d consider-using-in -d consider-using-dict-comprehension -d useless-import-alias -d consider-using-f-string --max-args=25 --max-branches=20 --max-statements=65 --min-public-methods=0 --max-public-methods=35 --min-similarity-lines=30 --max-line-length=120 (exited with code 28)
___________________________________________________________________________ summary ___________________________________________________________________________
ERROR:   analysis: commands failed

```

--- a/SoftLayer/CLI/environment.py
+++ b/SoftLayer/CLI/environment.py
@@ -17,7 +17,7 @@ import SoftLayer
 from SoftLayer.CLI import formatting
 from SoftLayer.CLI import routes
 
-# pylint: disable=too-many-instance-attributes, invalid-name, no-self-use
+# pylint: disable=too-many-instance-attributes, invalid-name
 
 # Calling pkg_resources.iter_entry_points shows a false-positive
 # pylint: disable=no-member

--- a/SoftLayer/auth.py
+++ b/SoftLayer/auth.py
@@ -5,7 +5,7 @@
 
     :license: MIT, see LICENSE for more details.
 """
-# pylint: disable=no-self-use
+
 
 __all__ = [
     'BasicAuthentication',

--- a/SoftLayer/managers/account.py
+++ b/SoftLayer/managers/account.py
@@ -12,7 +12,7 @@ from SoftLayer.exceptions import SoftLayerAPIError
 from SoftLayer import utils
 
 # Invalid names are ignored due to long method names and short argument names
-# pylint: disable=invalid-name, no-self-use
+# pylint: disable=invalid-name
 
 LOGGER = logging.getLogger(__name__)
 

--- a/SoftLayer/managers/cdn.py
+++ b/SoftLayer/managers/cdn.py
@@ -9,7 +9,7 @@ import SoftLayer
 from SoftLayer import utils
 
 
-# pylint: disable=no-self-use,too-many-lines,too-many-instance-attributes
+# pylint: disable=too-many-lines,too-many-instance-attributes
 
 
 class CDNManager(utils.IdentifierMixin, object):

--- a/SoftLayer/managers/dedicated_host.py
+++ b/SoftLayer/managers/dedicated_host.py
@@ -13,7 +13,7 @@ from SoftLayer.managers import ordering
 from SoftLayer import utils
 
 # Invalid names are ignored due to long method names and short argument names
-# pylint: disable=invalid-name, no-self-use
+# pylint: disable=invalid-name
 
 LOGGER = logging.getLogger(__name__)
 

--- a/SoftLayer/managers/email.py
+++ b/SoftLayer/managers/email.py
@@ -10,7 +10,7 @@ from SoftLayer import utils
 
 
 # Invalid names are ignored due to long method names and short argument names
-# pylint: disable=invalid-name, no-self-use
+# pylint: disable=invalid-name
 
 
 class EmailManager(utils.IdentifierMixin, object):

--- a/SoftLayer/managers/hardware.py
+++ b/SoftLayer/managers/hardware.py
@@ -20,7 +20,7 @@ from SoftLayer import utils
 LOGGER = logging.getLogger(__name__)
 
 # Invalid names are ignored due to long method names and short argument names
-# pylint: disable=invalid-name, no-self-use, too-many-lines
+# pylint: disable=invalid-name, too-many-lines
 
 EXTRA_CATEGORIES = ['pri_ipv6_addresses',
                     'static_ipv6_addresses',

--- a/SoftLayer/managers/ordering.py
+++ b/SoftLayer/managers/ordering.py
@@ -5,7 +5,7 @@
 
     :license: MIT, see LICENSE for more details.
 """
-# pylint: disable=no-self-use
+
 
 from re import match
 

--- a/SoftLayer/managers/storage.py
+++ b/SoftLayer/managers/storage.py
@@ -25,7 +25,7 @@ class StorageManager(utils.IdentifierMixin, object):
         self.client = client
         self.resolvers = [self._get_ids_from_username]
 
-    def _get_ids_from_username(self, username):  # pylint: disable=unused-argument,no-self-use
+    def _get_ids_from_username(self, username):  # pylint: disable=unused-argument
         """Should only be actually called from the block/file manager"""
         return []
 

--- a/SoftLayer/managers/vs.py
+++ b/SoftLayer/managers/vs.py
@@ -28,7 +28,7 @@ EXTRA_CATEGORIES = ['pri_ipv6_addresses',
                     'software_guard_extensions']
 
 
-# pylint: disable=no-self-use,too-many-lines
+# pylint: disable=,too-many-lines
 
 
 class VSManager(utils.IdentifierMixin, object):
@@ -360,7 +360,7 @@ class VSManager(utils.IdentifierMixin, object):
                     'prices': get_item_price(item['prices'], location_group_id)
                 })
 
-            elif category.__contains__('guest_disk'):
+            elif 'guest_disk' in category:
                 local_disk.append({
                     'name': item['description'],
                     'capacity': item['capacity'],
@@ -1043,7 +1043,7 @@ class VSManager(utils.IdentifierMixin, object):
             disk_number = 0
             vsi_disk = self.get_instance(instance_id)
             for item in vsi_disk.get('billingItem').get('children'):
-                if item.get('categoryCode').__contains__('guest_disk'):
+                if 'guest_disk' in item.get('categoryCode'):
                     if disk_number < int("".join(filter(str.isdigit, item.get('categoryCode')))):
                         disk_number = int("".join(filter(str.isdigit, item.get('categoryCode'))))
             for disk_guest in disk:

--- a/SoftLayer/managers/vs_capacity.py
+++ b/SoftLayer/managers/vs_capacity.py
@@ -14,7 +14,7 @@ from SoftLayer.managers.vs import VSManager
 from SoftLayer import utils
 
 # Invalid names are ignored due to long method names and short argument names
-# pylint: disable=invalid-name, no-self-use
+# pylint: disable=invalid-name
 
 LOGGER = logging.getLogger(__name__)
 

--- a/SoftLayer/managers/vs_placement.py
+++ b/SoftLayer/managers/vs_placement.py
@@ -11,7 +11,7 @@ import logging
 from SoftLayer import utils
 
 # Invalid names are ignored due to long method names and short argument names
-# pylint: disable=invalid-name, no-self-use
+# pylint: disable=invalid-name
 
 LOGGER = logging.getLogger(__name__)
 

--- a/SoftLayer/transports/__init__.py
+++ b/SoftLayer/transports/__init__.py
@@ -17,7 +17,7 @@ from .transport import SoftLayerListResult as SoftLayerListResult
 from .xmlrpc import XmlRpcTransport
 
 # transports.Request does have a lot of instance attributes. :(
-# pylint: disable=too-many-instance-attributes, no-self-use
+# pylint: disable=too-many-instance-attributes
 
 __all__ = [
     'Request',

--- a/SoftLayer/transports/transport.py
+++ b/SoftLayer/transports/transport.py
@@ -30,7 +30,7 @@ def get_session(user_agent):
 
 
 # transports.Request does have a lot of instance attributes. :(
-# pylint: disable=too-many-instance-attributes, no-self-use
+# pylint: disable=too-many-instance-attributes
 class Request(object):
     """Transport request object."""
 

--- a/tox.ini
+++ b/tox.ini
@@ -31,8 +31,6 @@ commands =
            -r n \
            --ignore=tests,fixtures \
            -d too-many-locals \
-           -d star-args \
-           -d redefined-variable-type \
            -d locally-disabled \
            -d no-else-return \ 
            -d len-as-condition \

